scripts/build-rpm

```diff
@@ -61,6 +61,12 @@ ARCH="$(uname -m)"
 DEST_DIR="/build/${DIST_ID}/${DIST_VERSION}/${ARCH}/"
 (
 	set -x
+	# Set 'runc_nokmem' build tag for RHEL/CentOS 7.x
+	if [[ "$DIST_ID" == "centos" || "$DIST_ID" == "rhel" ]]; then
+		if [[ "$DIST_VERSION" =~ ^7.* ]]; then
+			export RUNC_NOKMEM="nokmem"
+		fi
+	fi
```

Comment:
This script uses `/bin/sh`, and the code above is using Bashisms; can you change to;

```suggestion
	# Set 'runc_nokmem' build tag for RHEL/CentOS 7.x
	if [ "$DIST_ID" = "centos" ] || [ "$DIST_ID" = "rhel" ]; then
		case "$DIST_VERSION" in
		"7"*)
			export RUNC_NOKMEM="nokmem"
			;;
		esac
	fi
```

Reply:
Thanks for catching that (and applying the changes in #196)!

Reply:
Thank _you_ for bringing it to our attention 👍